lianad/src/payjoin/receiver.rs

```diff
@@ -115,7 +124,13 @@ fn identify_receiver_outputs(
     secp: &secp256k1::Secp256k1<secp256k1::VerifyOnly>,
 ) -> Result<(), Box<dyn Error>> {
     let proposal = proposal
-        .identify_receiver_outputs(|_| Ok(true))
+        .identify_receiver_outputs(|script| {
+            let address = bitcoin::Address::from_script(script, db_conn.network()).unwrap();
```

Comment:
Unwrap here is ok bc we need to do a larger error type refactor. But it can be trivially replaced with ? bc the error type in Box dyn error